docs/general/xs/functions.json

```diff
@@ -1100,19 +1100,19 @@
                     "name": "targetPlayerID",
                     "type": "int",
                     "required": false,
-                    "desc": "Check if the action is being performed on a unit (eg. attacking) of this player"
+                    "desc": "Check if the action is being performed on a unit (eg. attacking) of this player. Can use -1 to ignore this filter."
                 },
                 {
                     "name": "targetType",
                     "type": "int",
                     "required": false,
-                    "desc": "Check if the action is being performed on a unit of this type. Values 9xx refer to classes"
+                    "desc": "Check if the action is being performed on a unit of this type. Values 9xx refer to classes. Can use -1 to ignore this filter."
```

Comment:
tbh, when testing it never worked with 9xx class values, only with specific id.

Reply:
That's a bug, yes - its meant to work iirc